@here/harp-text-canvas/lib/rendering/TextStyle.ts

```diff
@@ -55,13 +55,38 @@ export enum VerticalAlignment {
 
 /**
  * Horizontal alignment to be used when placing text.
+ *
+ * @note Horizontal alignment value is exactly opposite to [[HorizontalPlacement]] value,
+ * cause when you place text on the right side of point (or icon) you normally want to have it
+ * left-aligned.
+ * @note Alignment attributes `hAlignment` and `vAlignment` are now depreciated in Theme
+ * definition for [[PoiTechinque]] and [[LineMarkerTechnique]] so use rather `hPlacement` and
+ * `vPlacement` for them.
  */
 export enum HorizontalAlignment {
     Left = 0.0,
     Center = -0.5,
     Right = -1.0
 }
 
+/**
+ * Vertical position of text area relative to the placement context (point, line).
+ */
+export enum VerticalPlacement {
```

Comment:
Same as VerticalAlignment. if you could make an alias, it'd be ok (but i don't you think you can do that in typescript). Otherwise I don't think it's worth to duplicate the enum.